src/actions/pipeline.js

```diff
@@ -0,0 +1,44 @@
+import { toastr } from 'react-redux-toastr';
+import {
+  GET_PIPELINES_CHANNEL,
+  GET_PIPELINE_CHANNEL,
+} from '../shared/channels';
+import { FLAG_GETTING_PIPELINES, FLAG_GETTING_PIPELINE } from '../shared/types';
+import { createFlag } from './common';
+import {
+  ERROR_MESSAGE_HEADER,
+  ERROR_GETTING_PIPELINES_MESSAGE,
+  ERROR_GETTING_PIPELINE_MESSAGE,
+} from '../shared/messages';
+
+// eslint-disable-next-line import/prefer-default-export
+export const getPipelines = () => (dispatch) => {
+  const flagGettingPipelines = createFlag(FLAG_GETTING_PIPELINES);
+  try {
+    dispatch(flagGettingPipelines(true));
+    // NOT IMPLEMENTED ON ELECTRON.JS
+    window.ipcRenderer.send(GET_PIPELINES_CHANNEL);
+    window.ipcRenderer.once(GET_PIPELINES_CHANNEL, async (_, payload) => {
```

Comment:
It's a convention when an event handler needs a function that needs multiple parameters but only some of them are used, so `_` in this case is an `event` but since I'm not using it I'm naming it `_`. I can stick to the conventions of `e` or `event`.